src/ui/Components/ConverterPanel.ts

```diff
@@ -35,6 +35,14 @@ export class Converter {
                 return str.substring(0, i);
         return str + "...";
     }
+
+    static unixToWinTs(wints: int): int {
```

Comment:
Despite the name of this function, it seems to convert not "Unix to Win", but actually vice versa: FILETIME to UNIX.

Reply:
Sure, I don't know what went wrong in my brain. Changed to a proper name.